maps/away/away_site/orion_automated_station/orion_automated_station.dm

```diff
@@ -0,0 +1,253 @@
+
+// -------------------------------- map and station defs
+
+/datum/map_template/ruin/away_site/orion_automated_station
+	name = "Orion Express Automated Station" // not visible ingame, but this should be unique for visibility purposes
+	description = "Orion Express Automated Station" // not visible ingame
+	unit_test_groups = list(3)
+
+	id = "orion_automated_station" // arbitrary tag to make things work, this should be lowercase and unique
+	spawn_cost = 1
+	spawn_weight = 1
+	suffixes = list("away_site/orion_automated_station/orion_automated_station.dmm")
+
+	sectors = list(ALL_POSSIBLE_SECTORS)
+	template_flags = TEMPLATE_FLAG_SPAWN_GUARANTEED
+
+/singleton/submap_archetype/orion_automated_station // arbitrary duplicates of the above name/desc
+	map = "Orion Express Automated Station"
+	descriptor = "Orion Express Automated Station"
+
+/obj/effect/overmap/visitable/sector/orion_automated_station // this is the actual overmap object that spawns at roundstart
+	name = "Orion Express Automated Station (replaced in /New())" // this and desc is visible ingame when the object is scanned by any scanner
+	desc = "\
+		Orion Express Automated Station, common sight all around the Spur. \
+		But especially so in the outer edges of the Coalition and the Frontier, \
+		where the Orion Express service depends on ordinary people and ships picking up and delivering packages for each other, \
+		with Orion Express only delivering to the automated stations and other distribution points. \
+		This particular station is of the smaller variety, with very few facilities.\
+		"
+	icon_state = "ox_auto_station"
+
+	static_vessel = TRUE
+	generic_object = FALSE
+	icon = 'icons/obj/overmap/overmap_stationary.dmi'
+	icon_state = "ox_auto_station"
+	color = "#bbb186"
+	designer = "Unknown"
+	volume = "26 meters length, 58 meters beam/width, 20 meters vertical height"
+	weapons = "Not apparent"
+	sizeclass = "Industrial Station"
+
+	initial_generic_waypoints = list(
+		"nav_ox_auto_station_dock_west",
+		"nav_ox_auto_station_dock_east",
+		"nav_ox_auto_station_dock_north_1",
+		"nav_ox_auto_station_dock_north_2",
+		"nav_ox_auto_station_dock_south_1",
+		"nav_ox_auto_station_dock_south_2",
+		"nav_ox_auto_station_catwalk_north",
+		"nav_ox_auto_station_catwalk_south",
+		"nav_ox_auto_station_space_north_east",
+		"nav_ox_auto_station_space_north_west",
+		"nav_ox_auto_station_space_south_east",
+		"nav_ox_auto_station_space_south_west",
+	)
+
+/obj/effect/overmap/visitable/sector/orion_automated_station/New()
+	designation = "[pick("Orion Express Automated Station #6282", "Orion Express Automated Station #1112", "Orion Express Automated Station #0091", "Orion Express Automated Station #9123", "Orion Express Automated Station #0157", "Orion Express Automated Station Zeta", "Orion Express Automated Station Kilo", "Orion Express Automated Station Uniform", "Orion Express Automated Station Whiskey")]"
```

Comment:
Wouldn't it be better to have it as a thing of picking a number from 0001 to 9999 rather then these particular arbitrary numbers?

Reply:
done